docs/en/reference/improving-performance.rst

```diff
@@ -61,8 +65,9 @@ Apply Best Practices
 A lot of the points mentioned in the Best Practices chapter will
 also positively affect the performance of Doctrine.
 
+See :doc:`Best Practices <best-practices>`
 
 Change Tracking policies
 ------------------------
 
-See: :doc:`Change Tracking Policies <reference/change-tracking-policies>`
+See: :doc:`Change Tracking Policies <change-tracking-policies>`
```

Comment:
The folder name ist actually not needed.

Reply:
Oh wow that's weird… what if 2 files in different directories have the same name?

Reply:
As long as we don't know why it works, we should keep the folder name like in the other links.

Reply:
Ok, I reverted that change and adjusted the other link.